x-pack/filebeat/input/gcppubsub/config.go

```diff
@@ -36,6 +36,8 @@ type config struct {
 
 	// JSON blob containing authentication credentials and key.
 	CredentialsJSON []byte `config:"credentials_json"`
+
+	PubsubAlternativeHost string `config:"pubsub_alternative_host"`
```

Comment:
I'm debating about the `pubsub` prefix. I thinking to remove it since the package is called pubsub and the in config it will be part of the gcp-pubsub input.

```suggestion
	// Overrides the default Pub/Sub service address and disables TLS. For testing.
	AlternativeHost string `config:"alternative_host"`
```